tests/README.md

```diff
@@ -1,25 +1,239 @@
-Mender Integration testing
-===================
+# Mender Integration Testing
 
+<!-- markdown-toc start - Don't edit this section. Run M-x markdown-toc-refresh-toc -->
+**Table of Contents**
 
-####Requirements
+- [Mender Integration Testing](#mender-integration-testing)
+    - [Getting Started](#getting-started)
+    - [Installing Dependencies](#installing-dependencies)
+        - [Debian](#debian)
+        - [Alpine Linux](#alpine-linux)
+        - [Python2 and Python3](#python2-and-python3)
+    - [Running the Tests Locally](#running-the-tests-locally)
+        - [Isolating Python Dependencies](#isolating-python-dependencies)
+            - [Install](#install)
+            - [Initialize the Virtual Environment](#initialize-the-virtual-environment)
+                - [(Optional) -- Select Python Version](#optional----select-python-version)
+            - [Activate the Virtual Environment](#activate-the-virtual-environment)
+            - [Install the Python Dependencies from the Requirements File](#install-the-python-dependencies-from-the-requirements-file)
+        - [Running the Tests](#running-the-tests)
+    - [Modifying the Docker Images Employed](#modifying-the-docker-images-employed)
+        - [Example -- Running with a Custom Backend Service](#example----running-with-a-custom-backend-service)
+        - [Example -- Running with a Custom Client](#example----running-with-a-custom-client)
+    - [Running the Tests in a VM](#running-the-tests-in-a-vm)
+        - [Vagrant](#vagrant)
+- [Legacy Stuff (TODO -- Remove)](#legacy-stuff-todo----remove)
+    - [-](#--1)
+    - [OS X](#os-x)
+    - [Known issues](#known-issues)
+    - [Tips and Tricks](#tips-and-tricks)
+
+<!-- markdown-toc end -->
+
+
+-------------------------------------------------------------------------------
+
+
+## Getting Started
+
+The dependencies for the integration tests are collected and organized in
+dependency files in the `./requirements` folder, and separated into
+
+
+> These requirement files are using the *hashicorp/bionic64* *Vagrant* image is
+> as the basis for the requirements files generated.
+
+| Debian | Alpine | Python |
+| ------------- | :-------------: | -----: |
+| *deb-requirements.txt* | *apt-requirements.txt* | *python-requirements.txt* |
+
+## Installing Dependencies
+
+### Debian
+
+```bash
+$ apt install -yyq $(cat requirements/integration/deb-requirements.txt)
+```
+
+### Alpine Linux
+
+```bash
+# TODO -- What is the correct command for the Alpine package manager
+$ apk add -yyq $(cat requirements/integration/apk-requirements.txt)
```

Comment:
I think is just `apk --update add`. Optionally, pass the `--no-cache` flag: `apk --update --no-cache add`